src/main/kotlin/dev/stalla/util/ItunesCategoryExtensions.kt

```diff
@@ -0,0 +1,20 @@
+package dev.stalla.util
+
+import dev.stalla.model.itunes.ItunesCategory
+import dev.stalla.model.itunes.NestedItunesCategory
+import dev.stalla.model.itunes.SimpleItunesCategory
+
+@InternalApi
+internal val itunesCategoryMap: Map<String, ItunesCategory> = run {
+    // Note that the map calls here do a necessary smart cast
+    val categories = listOf(
+        SimpleItunesCategory.values().map { it },
+        NestedItunesCategory.values().map { it }
```

Comment:
Enums have the one major benefit of providing the `.values()` method. If we went with object instances of sealed classes, we'd have to maintain these lists ourself, or use reflection.